.travis.yml

```diff
@@ -1,82 +1,106 @@
 # https://travis-ci.org/tornadoweb/tornado
 
+os: linux
 dist: xenial
 language: python
 addons:
   apt:
     packages:
       - libgnutls-dev
 
-# For a list of available versions, run
-#     aws s3 ls s3://travis-python-archives/binaries/ubuntu/16.04/x86_64/
+env:
+  global:
+    - CIBW_BUILD="cp3[5678]*"
+    - CIBW_TEST_COMMAND="python3 -m tornado.test"
+    - CIBW_TEST_COMMAND_WINDOWS="python -m tornado.test --fail-if-logs=false"
+
+# Before starting the full build matrix, run one test configuration
+# and the linter (the `black` linter is especially likely to catch
+# first-time contributors).
+stages:
+  - quick
+  - test
+
 jobs:
   fast_finish: true
   include:
+    # We have two and a half types of builds: Wheel builds run on all supported
+    # platforms and run the basic test suite for all supported python versions.
+    # Sdist builds (the "and a half") just build an sdist and run some basic
+    # validation. Both of these upload their artifacts to pypi if there is a
+    # tag on the build and the key is available.
+    #
+    # Tox builds run a more comprehensive test suite with more configurations
+    # and dependencies (getting all these dependencies installed for wheel
+    # builds is a pain, and slows things down because we don't use as much
+    # parallelism there. We could parallelize wheel builds more but we're also
+    # amortizing download costs across the different builds). The wheel builds
+    # take longer, so we run them before the tox builds for better bin packing
+    # in our allotted concurrency.
+    - python: '3.8'
+      arch: amd64
+      services: docker
+      env: BUILD_WHEEL=1
+    - python: '3.8'
+      arch: arm64
+      services: docker
+      env: BUILD_WHEEL=1  ASYNC_TEST_TIMEOUT=15
+    - os: windows
+      env: PATH=c/Python38:/c/Python38/Scripts:$PATH BUILD_WHEEL=1
```

Comment:
it's odd that the first PATH item here starts with `c/` while the second one starts with `/c/`

Reply:
Yep, that was my mistake copying from the cibuildwheel docs. Seems to work either way, though?

Reply:
I think the reason why it worked, is because in the windows case you never call "python" directly, you call "cibuildwheel" or "pip" etc, which are in the `Scripts/` dir, and these things have an embedded path the python they were installed with.